frontend/src/SignupPage/SignupPage.jsx

```diff
@@ -42,7 +42,7 @@ class SignupPage extends React.Component {
         this.setState({ isLoading: false, signupSuccess: true });
       },
       () => {
-        toast.error('Invalid email', {
+        toast.error('Email already exists', {
```

Comment:
I think we need to display the error message in the response from server. The request can fail for many reasons.

Reply:
> I think we need to display the error message in the response from the server. The request can fail for many reasons.

Yeah, but how'd you check the error tho? I am not sure how to go about this, let me know if you have any suggestions

Reply:
`server/src/services/auth.service.ts:61` - you might need to add some logic here to respond with a meaningful message if the email already exists.